designsafe/static/scripts/projects/components/manage-experiments/manage-experiments.component.js

```diff
@@ -16,10 +16,15 @@ class ManageExperimentsCtrl {
     $onInit() {
         this.project = this.resolve.project;
         this.edit = this.resolve.edit;
-        
+
+        let remData = (data) => {
+            data.atlss = data.atlss.filter(a => a.name != "hybrid_simulation"); 
```

Comment:
sanity check - will `data.atlss` always exist and be an array?

Reply:
Yep, that "atlss" field is in a JSON object that was setup as a set of input field mappings for a form. This is the JSON that it's referencing.
https://github.com/DesignSafe-CI/portal/blob/master/designsafe/static/scripts/projects/components/manage-experiments/experimental-data.json